specifyweb/frontend/js_src/lib/components/Header/NotificationRenderers.tsx

```diff
@@ -119,6 +123,30 @@ export const notificationRenderers: IR<
       />
     );
   },
+  'record-merge-starting'() {
+    return mergingText.mergingHasStarted();
+  },
+  'record-merge-failed'() {
+    return mergingText.mergingFailed();
+  },
+  'record-merge-succeeded'(notification) {
+    const id = parseInt(notification.payload.new_record_id);
+    const tableName = notification.payload.table;
+    const model = getModel(tableName);
+    const resource =
+      typeof model === 'object' ? new model.Resource({ id }) : undefined;
+    return (
+      resource !== undefined && (
+        <>
+          {mergingText.mergingSucceeded()}
+          <Link.NewTab href={`/specify/view/${tableName}/${id}`}>
+            <TableIcon label name={tableName} />
+            <FormattedResource asLink={false} resource={resource} />
```

Comment:
`FormattedResource` as a link already opens the resource in a new tab. Why did you need to wrap it in a `Link.NewTab` and set the `FormattedResource` to not be a link. 
Was it it be consistent with other Notification links?

Reply:
No you are right. I didn't have FormattedResource at first and didn't remove it once I implemented it